src/bots/filler.ts

```diff
@@ -1121,28 +1127,6 @@ export class FillerBot implements Bot {
 			return false;
 		}
 
-		const perpMarket = this.driftClient.getPerpMarketAccount(
-			nodeToFill.node.order.marketIndex
-		)!;
-		// if making with vAMM, ensure valid oracle
-		if (
-			nodeToFill.makerNodes.length === 0 &&
-			!isVariant(perpMarket.amm.oracleSource, 'prelaunch')
-		) {
-			const oracleIsValid = isOracleValid(
-				perpMarket,
-				oraclePriceData,
-				this.driftClient.getStateAccount().oracleGuardRails,
-				this.getMaxSlot()
-			);
-			if (!oracleIsValid) {
-				logger.error(
-					`Oracle is not valid for market ${marketIndex}, skipping fill with vAMM`
-				);
-				return false;
-			}
-		}
-
```

Comment:
it will fail in sim already and the oracle crank is meant to update the oracle account if the price is stale

Reply:
it will fail in sim if it's truly stale, and then get throttled. and we shouldn't be awaiting the results of the sim anyways (we fire and forget)- this seems redundant to me.